libraries/AP_AHRS/AP_AHRS.h

```diff
@@ -573,6 +573,16 @@ class AP_AHRS
         return _rsem;
     }
 
+    // convert a vector from body to earth frame
+    Vector3f body_to_earth(const Vector3f &v) const {
```

Comment:
as discussed on the dev call let's move this higher to be in the same place in the header as rotate_earth_to_body2D

Reply:
I've dropped the rotate_ prefix on the 2D calls